packages/reactotron-server/src/app/system-config.ts

```diff
@@ -0,0 +1,21 @@
+import { ReactotronApp } from "reactotron-core-ui"
+import { ApiResponseTimeline, ApiResponsePreview } from "./commands/ApiResponse"
+import { SagaTaskTimeline, SagaTaskPreview } from "./commands/SagaTaskComplete"
+
+export function registerSystemTimelineCommands(app: ReactotronApp) {
```

Comment:
This particular file could import the ReactotronApp directly but I wanted to emulate it more like what the plugins will end up with. One day hopefully most of the commands that end up in here will be in a plugin (looking at you `saga.task.complete`)